Sources/Provider.swift

```diff
@@ -0,0 +1,92 @@
+import URI
+import Vapor
```

Comment:
How is Vapor being included if this package doesn't depend on it? The `mongo-driver`s job should be mapping Fluent queries to Mongo. The `mongo-provider` is responsible for implementing the `Vapor.Provider.`